src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -109,35 +125,31 @@ export default class EthereumBlockchainService implements BlockchainService {
    * previous attempt's gas, even if the gas price on chain has gone down since then. This is because
    * retries of the same transaction (using the same nonce) need to have a gas price at least 10% higher
    * than any previous attempts or the transaction will fail.
-   * @param currentGasEstimate
+   * @param feeData
    * @param attempt
    * @param previousGas
    */
   static increaseGasPricePerAttempt(
-    currentGasEstimate: BigNumber,
+    feeData: providers.FeeData,
     attempt: number,
     previousGas: BigNumberish | undefined
   ): BigNumber {
-    const tenPercent = currentGasEstimate.div(10)
-    const additionalGas = tenPercent.mul(attempt)
-    const newGas = currentGasEstimate.add(additionalGas)
+    // Ganache still does not support 1559, so here is a workaround
```

Comment:
see above comment.  Would prefer if we could handle this in test code instead of prod code if possible

Reply:
Disagree. For debugging it is helpful to have ganache support. But maybe in that case we would override?

However still think backwards compatibility should be supported in general

Reply:
Fair. In that case then I agree with you that we should do more to separate out the logic for 1559 transactions from the logic for legacy style transactions so it's clear what use case each piece of code is used for.